syllabus.html

```diff
@@ -0,0 +1,575 @@
+                project-based learning. Students complete eight projects encapsulating the program’s topics, culminating
+                in one final capstone project where they design and build a remote-controlled rover.
+            </p>
+            <h1 class="fw-bold mb-3">
+                Our Mission
+            </h1>
+            <p class="mb-3">
+                We have two goals: One is to provide students with the fundamental skills they need to be
+                competitive candidates for their first engineering jobs and internships. Open Project Space accomplishes
+                this with regular workshops and lectures that complement the projects. We train participants to use
+                soldering equipment, design printed circuit boards, and read technical manuals and specifications, none
+                of which are skills acquired from traditional coursework. Our curriculum also covers circuit analysis,
+                programming, and embedded design concepts to provide students with an early overview of topics they
+                would
+                otherwise learn much later in their degree program.
+            </p>
+            <p class="mb-5">
+                The second goal is to support and motivate students in a challenging degree pathway.
+                We offer one-on-one support both in-person, through regular lab sessions, and in online forums.
+                We implore students to ask probing
+                questions during our lectures and workshops so that they can solidify their understanding of the
+                course material.
+            </p>
+
+            <h1 class="fw-bold mb-3">
+                Enrollment Requirements
+            </h1>
+            <p class="mb-5">
+                Undergraduate students who attend the University of California, Irvine 
+                are eligible to apply for course enrollment. The deadline for application submissions is 
+                <strong>this date</strong>. Students who receive a course acceptance offer must pay the course deposit and fee before enrollment.
+            </p>
+
+            <h1 class="fw-bold mb-3">
+                Course Deposit + Fee
+            </h1>
+            <p class="mb-5">
+                You will pay a <strong>deposit</strong> of <strong>$60</strong> and
+                a <strong>non-refundable course fee</strong> of <strong>$60</strong> ($120 total) at the beginning of
+                the program.
+                In exchange, you will receive a parts kit with the components needed to complete the course projects.
+                Upon completing all of the required projects at the end of the academic year, you will be reimbursed the
+                deposit.
+            </p>
+        </section>
+
+        <!-- Section: Timeline -->
+        <section class="pb-5">
+            <h1 class="fw-bold mb-5">
+                Schedule
```

Comment:
Maybe schedule can just be a jpg? Lot of scrolling for the information. Maybe instead of vertical do horizontal (and each column a quarter?) Might be overthinking.